### PR TITLE
SAPHana: set_SRHOOK_PRIM 

### PR DESCRIPTION
**Pull request by fmherschel**
- set site specific attributes, if already used in the past (bsc#1205535)

### CODE REVIEW

**Comment by fmherschel** [comment]
@angelabriel : wegen des PR - ggf erst nochmal reden dann mergen :)

File: ra/SAPHana [hunk 1]
```diff
@@ -2819,12 +2838,7 @@ function saphana_promote_clone() {
     fi
     local lastSrHook
```
Comment (angelabriel) [inline ra/SAPHana:2839]: no longer needed here. Better move to function 'set_SRHOOK_PRIM'

**Review by angelabriel — COMMENTED** [review]
Looks so nice and simple :smile: 
Good idea to depend on the remote attribute too.

**Comment by fmherschel** [comment]
@angelabriel You mean like this? [488f7aa](https://github.com/SUSE/SAPHanaSR/pull/157/commits/488f7aae45c22c07f414569ea2b17c13338efa23)

**Comment by angelabriel** [comment]
> @angelabriel You mean like this? [488f7aa](https://github.com/SUSE/SAPHanaSR/pull/157/commits/488f7aae45c22c07f414569ea2b17c13338efa23)

yes, absolutely

**Review by angelabriel — APPROVED** [review]

**Pull request merged**